lib/executor/ramping_vus.go

```diff
@@ -107,62 +107,66 @@ func (vlvc RampingVUsConfig) Validate() []error {
 // execution steps would look something like this:
 //
 // VUs  ^
-//    10|          *
-//     9|         ***
-//     8|        *****
-//     7|       *******
-//     6|      *********
-//     5|     ***********
-//     4|    *************
-//     3|   ***************
-//     2|  *****************
-//     1| *******************
-//     0------------------------> time(s)
-//       01234567890123456789012   (t%10)
-//       00000000001111111111222   (t/10)
+//
+//	10|          *
+//	 9|         ***
+//	 8|        *****
+//	 7|       *******
+//	 6|      *********
```

Comment:
This is off the `^` is the end of the graph :(

Reply:
will fix it

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/fcdcb2237c5a91a51c406d6bb80d27db592c4ec6..07630ed366716acf6b5e0675946301188d3b23a9)